mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1030,6 +1030,15 @@ impl<'n> Nexus<'n> {
                 })))
                 .await;
         }
+        // If we are faulted then rather than failing all IO back to the
+        // initiator we can instead leave the subsystem paused, and wait
+        // for the control-plane to do something about this.
+        // Meanwhile the initiator will begin it's reconnect loop and won't see
```

Comment:
Is this reconnect loop going to exponentially backoff till the nexus is paused?

Reply:
No, it's every 10s after each connect failure, until a nexus is available again.